freqtrade/utils.py

```diff
@@ -45,8 +45,15 @@ def start_trading(args: Dict[str, Any]) -> int:
     """
     from freqtrade.worker import Worker
     # Load and run worker
-    worker = Worker(args)
-    worker.run()
+    try:
+        worker = Worker(args)
```

Comment:
`worker = None` should be assigned above, otherwise `if worker` will fail below if an exception happens during the initialization of worker